requestmanager/server.go

```diff
@@ -233,7 +233,7 @@ func (rm *RequestManager) cancelRequest(requestID graphsync.RequestID, onTermina
 	if !ok {
 		if onTerminated != nil {
 			select {
-			case onTerminated <- graphsync.RequestNotFoundErr{}:
+			case onTerminated <- &graphsync.RequestNotFoundErr{}:
```

Comment:
generally I prefer not to use pointer types for errors -- they're immutable values. any reason you converted it? I see you are working with errors.Is but I don't think you should need that to make it work.

Reply:
I had trouble making `errors.As` working without pointers, but Dan helped me figure that one out; so this is reverted